guide/getting-started/hardware.md

```diff
@@ -68,13 +69,14 @@ You may already be familiar with key fobs from your bank or work. They are small
    height = 400
 %}
 
-Hardware wallets for bitcoin play a similar role but with several differences. Like key fobs, they securely store an authentication mechanism and isolate it from other devices and networks. In bitcoin, these are the seed, associated keys, and all functions that require direct interaction with them.
+Bitcoin hardware wallets play a similar role but with several noteworthy differences. Like key fobs, they securely store an authentication method and isolate it from the Internet, networks, and other devices. In bitcoin, these are the seed, associated keys, and functions that require interaction with them.
 
-Because hardware wallets are so optimized for this specific use case, they have tiny screens and limited input options. Most interactions with them happen via desktop software that the manufacturer also provides. Users perform most wallet interactions on their desktops, and the application sends only specific requests to the hardware wallet to verify or sign data. This provides a good balance of security and convenience and keeps hardware wallets generally affordable for most use cases.
+Because hardware wallets are so optimized for this specific use case, they have tiny screens and limited input options. Most interactions with them happen via desktop software that the manufacturer provides. The application sends only specific requests to the hardware wallet to verify or sign data. This provides a strong balance of security and convenience and keeps hardware wallets generally affordable.
```

Comment:
```suggestion
Because hardware wallets are so optimized for this specific use case, they have tiny screens and limited input options. Most interactions with them happen via desktop software that the manufacturer provides. The application sends only specific requests to the hardware wallet to verify or sign data. This provides a strong balance of security and convenience.
```

Not sure why this makes them more affordable?

Reply:
Agreed "affordable" also depends on who's looking at it. Removed.